R/FilterState.R

```diff
@@ -2156,19 +2156,19 @@ DateFilterState <- R6::R6Class( # nolint
       fluidRow(
         div(
           actionButton(
+            class = "start_date_rest_button",
```

Comment:
should start and end data have `reset` not `rest`

Reply:
thanks